config/src/config/consensus_config.rs

```diff
@@ -123,14 +171,15 @@ impl Default for ConsensusConfig {
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 72,
                     max_sending_block_txns_override: 200,
-                    max_sending_block_bytes_override: 50 * 1024,
+                    // stop reducing size, so 100k transactions can still go through
+                    max_sending_block_bytes_override: 100 * 1024,
                 },
                 ChainHealthBackoffValues {
                     backoff_if_below_participating_voting_power_percentage: 69,
                     // in practice, latencies make it such that 2-4 blocks/s is max,
                     // meaning that most aggressively we limit to ~200-400 TPS
                     max_sending_block_txns_override: 100,
-                    max_sending_block_bytes_override: 25 * 1024,
+                    max_sending_block_bytes_override: 100 * 1024,
```

Comment:
I just realize we was limiting it too aggressively, in practice it needs to be bigger than the single maximum txn size (64kb today)

Reply:
yeah, I caught this for module publishing